lib/utils/base64u.js

```diff
@@ -36,3 +36,22 @@ export const encode = string => urlEncode(
 )
 
 export const match = base64u => /^[A-Za-z0-9\-_]+$/.test(base64u)
+
+/**
+ * Attempst to decode a passed string if recognized as base64u. Otherwise
+ * returns untouched input.
+ *
+ * @param  {String} base64u Possibly base64u encoded string
```

Comment:
Wow, JSDoc pioneering in this repo – I feel home :) Documentation standards could be a good topic for a team meeting soon.